internal/testutils/test_container_utils.go

```diff
@@ -164,5 +166,5 @@ func tryDetectPodmanRunning() bool {
 	if err != nil {
 		return false
 	}
-	return info.Host.MachineState == "Running"
+	return info.Host.MachineState == "Running" || strings.Contains(os.Getenv("DOCKER_HOST"), "podman.sock")
```

Comment:
I believe worth noting that this addition leverage a podman configuration which is not "docker compatibility".

Reply:
I don't follow, this function is precisely checking if podman is being used, right?

Reply:
The reason for the OR is on Linux systems that do not need a machine to run containers.